pallets/ds-maps/src/lib.rs

```diff
@@ -0,0 +1,192 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use frame_support::{
+    codec::{Decode, Encode},
+    decl_error, decl_event, decl_module, decl_storage, dispatch, ensure,
+    weights::{Weight},
+    Parameter,
+};
+use frame_system::ensure_signed;
+use pallet_ds_accounts as accounts;
+use accounts::REGISTRAR_ROLE;
+
+mod default_weight;
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
+pub enum ZoneType {
+    /// Forbidden type zone
+    Red,
+    /// Available for safe flights
+    Green,
+    /// Owns zones
+    Parent,
+}
+
+impl Default for ZoneType {
+    fn default() -> Self {
+        ZoneType::Green
+    }
+}
+
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, Default, Debug, PartialEq, Eq)]
+pub struct Point3D<Coord> {
+    x: Coord,
+    y: Coord,
+    z: Coord,
```

Comment:
Why x/y/z, not lat/lon/alt? I thought we're going to use geo coordinates.

Reply:
Sure, my bad.